src/javascripts/ng-admin/Crud/delete/BatchDeleteController.js

```diff
@@ -12,17 +12,14 @@ define(function () {
         this.view = view;
         this.entity = view.getEntity();
         this.entityIds = $state.params.ids;
-        this.selection = []; // fixme: query db to get selection
-        this.title = view.title();
-        this.description = view.description();
-        this.actions = view.actions();
-        this.loadingPage = false;
-        this.fields = view.fields();
+        this.alreadyConfirm = false;
```

Comment:
For more consistency with other attributes, can you provide a getter/setter function?